docs/monthly-meeting/2021-04-15.md

```diff
@@ -0,0 +1,88 @@
+# JupyterHub and BinderHub Team Meeting
+
+- **Date:** Thursday 15th April 2021
+- **Time:** 9AM Europe/London
+  - **Your timezone:** https://arewemeetingyet.com/London/2021-04-15/09:00/Hubs-Team-Meeting
+- **This HackMD:** https://hackmd.io/@sgibson91/hubs-team-meeting
+- **GitHub issue:** https://github.com/jupyterhub/team-compass/issues/
+- **Calendar for future meetings:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html
```

Comment:
```suggestion
  - **Your timezone:** <https://arewemeetingyet.com/London/2021-04-15/09:00/Hubs-Team-Meeting>
- **This HackMD:** <https://hackmd.io/@sgibson91/hubs-team-meeting>
- **GitHub issue:** <https://github.com/jupyterhub/team-compass/issues/>
- **Calendar for future meetings:** <https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html>
```

These URLs aren't hyperlinked in the RTD preview
![image](https://user-images.githubusercontent.com/1644105/115043992-f8d29780-9ecc-11eb-87d4-a0722711918f.png)
though the ones lower down in the notes surrounded by `<...>` are. Could try this?